### PR TITLE
Add longer timeouts for Galley tests.

### PR DESCRIPTION
**Pull request by nmittler**
Addresses #11464

### CODE REVIEW

File: galley/pkg/testing/events/events.go [hunk 1]
```diff
@@ -23,12 +23,14 @@ import (
 )
 
 const (
-	DefaultTimeout = 2 * time.Second
-	DefaultPeriod  = 100 * time.Millisecond
+	defaultSuccessTimeout = 5 * time.Second
+	defaultFailureTimeout = 2 * time.Second
```
Comment (ozevren) [inline galley/pkg/testing/events/events.go]: Should these be even longer? Especially the success case?

Comment (nmittler) [reply galley/pkg/testing/events/events.go]: suggestions? 10 sec?

Comment (ozevren) [reply galley/pkg/testing/events/events.go]: 3 second for failure, 10 second for success?

Comment (nmittler) [reply galley/pkg/testing/events/events.go]: done.

**Comment by nmittler** [comment]
@wenchenglu mind merging?

**Pull request merged**